main.scss

```diff
@@ -5,174 +5,194 @@
 @import "o-grid/main";
 @import "o-hoverable/main";
 
-$o-ft-footer-spacing-unit: 20px;
+/// Silent mode
+/// @type Bool
+$o-footer-is-silent: true !default;
 
-@include oFontsInclude(BentonSans);
+$o-footer-spacing-unit: 20px;
 
 // TODO: Remove this when a default shared brand color is defined in o-colors
-@include oColorsSetUseCase(o-ft-footer-brand, border, claret);
-
-@include oColorsSetUseCase(o-ft-footer, background, grey-tint5);
-@include oColorsSetUseCase(o-ft-footer, text, white);
-@include oColorsSetUseCase(o-ft-footer__pearson, background, grey-tint4);
-@include oColorsSetUseCase(o-ft-footer__divider, border, white);
-@include oColorsSetUseCase(o-ft-footer-item, text, white);
-
-.o-ft-footer {
-	@include oColorsFor(o-ft-footer, text background);
-	@include oFtTypographyFontSize(14);
-
-	margin-top: 40px;
-	padding: 0;
-	border-top: 5px solid oColorsGetColorFor(o-ft-footer-brand brand, border);
-	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
-	line-height: $o-ft-footer-spacing-unit;
-
-	> nav {
-		margin-top: $o-ft-footer-spacing-unit;
-		margin-bottom: $o-ft-footer-spacing-unit;
-		padding-bottom: $o-ft-footer-spacing-unit;
-		border-bottom: oColorsGetColorFor(o-ft-footer__divider, border) dotted 1px;
-	}
+@include oColorsSetUseCase(o-footer-brand, border, claret);
```

Comment:
It should be like this: https://github.com/Financial-Times/o-ft-header/blob/master/src/scss/_variables.scss#L5